hs_modflow_modelinstance/models.py

```diff
@@ -589,26 +589,13 @@ def metadata(self):
         md = MODFLOWModelInstanceMetaData()
         return self._get_metadata(md)
 
-    def has_required_content_files(self):

```

Comment:
This was removed to add more flexibility for this resource type. In the old implementation, It was required from a user to uploaded the .nam file at the beginning (this .nam file includes all the index for the required input files to has a complete instance uploaded for the MODFLOW model). Then the system loops through this .nam file and makes sure that all the input files are uploaded before making the resource public or discoverable. That is why we have this function, has_required_content_files to override the one in the hs_core/models.py. We got more than one suggestion about removing these limitations and restrictions, especially, because the .nam file may have different extensions forms or even without any extension at all. The most two popular extensions are: .nam, for the regular MODFLOW users, and .mfn, for the MODFLOW user under GMS. Now, rather than making these limitations and restriction the system tries to find one of these file extensions. If one of these extensions is uploaded the system starts to loop through and gives recommendations about any missing input files. However, if this .nam file has any other extension, a user will still be able to make the resource public or discoverable. Now, we do not need this function anymore and we can use the original one in hs_core/models.py.